src/main/java/gregtech/api/recipes/logic/ParallelLogic.java

```diff
@@ -383,7 +383,7 @@ protected static int getMaxRatioFluid(@Nonnull Map<FluidKey, Integer> countFluid
             int available = 0;
             // For every fluid gathered from the fluid inputs.
             for (Map.Entry<FluidKey, Integer> inputFluid : countFluid.entrySet()) {
-                // Strip the Non-consumable tags here, as FluidKey compares the tags, which causes finding matching fluids
+                // Strip the Non-consumable tags here, as FluidStorageKey compares the tags, which causes finding matching fluids
```

Comment:
This change is not correct, you are not using a FluidStorageKey here. You are still using a FluidKey